src/libraries/System.Net.Sockets/tests/FunctionalTests/Connect.cs

```diff
@@ -109,6 +109,7 @@ public async Task Connect_DualMode_MultiAddressFamilyConnect_RetrievedEndPoints_
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/54677", TestPlatforms.Linux)]
```

Comment:
@antonfirsov can you update the test so both `IPAddress.Loopback.MapToIPv6()` and `IPAddress.IPv6Loopback` are allowed for `localEndPoint.Address`.

I expect for `remoteEndPoint.Address` we should stick to `IPAddress.Loopback.MapToIPv6()` only because the test server is bound to `IPAddress.Loopback`.

Reply:
You are right. I overlooked the exception message, and for some reason thought LocalEndPoint was IPv4, not `::1`.
Makes sense to fix the test instead of disabling.